### PR TITLE
CVC4 / Yices2 as part of CI

### PR DESCRIPTION
**Pull request by nishantjr**

### CODE REVIEW

**Comment by nishantjr** [comment]
Note that buildbot needs to use `./build test-all` for this to run `smtTest`

File: build [hunk 1]
```diff
@@ -82,21 +112,27 @@ build_test_all() {
...
-    test)     build_test     "$@" ;;
-    test-all) build_test_all "$@" ;;
+    clean)          build_clean    "$@" ;;
+    deps)           build_deps     "$@" ;;
+    maude)          build_maude    "$@" ;;
+    maude-cvc4)     build_maude_cvc4 "$@" ;;
+    maude-yices2)   build_maude_yices2 "$@" ;;
+    tangle)         build_tangle   "$@" ;;
+    test)           build_test     "$@" ;;
+    test-all)       build_test_all "$@" ;;
```
Comment (ehildenb) [inline build]: Please line up the `$@` and make sure that all options here are documented below.

**Review by ehildenb — CHANGES_REQUESTED** [review]
Looks good, minor changes.

I would also like a `./build test-smt` option which ran just the SMT tests. `./build test-all` can call that function directly.

**Review by ehildenb — APPROVED** [review]

**Comment by ehildenb** [comment]
@nishantjr I've updaded Buildbot to include a step for running the SMT tests as well.

I prefer to keep it separated out into build steps so it's easier to see which step Buildbot fails on.

**Comment by ehildenb** [comment]
A small formatting error caused a weird bug in `./configure`, but it should be fixed now. Fingers crossed!

**Pull request merged**